server_extensions_api/src/main/java/com/generalbytes/batm/server/extensions/IExtensionContext.java

```diff
@@ -578,6 +578,11 @@ IIdentity updateIdentity(String identityId, String externalId, int state, int ty
      */
     ILocationDetail getLocationByPublicId(String locationPublicId);
 
+    /**
+     * Gets a location identified by External Location ID
+     */
```

Comment:
Could you also mention @param and @return in javadoc as in the other methods ?

Reply:
ok I did to keep it consistent, but in cases like this I often don't know what to write in there more than the parameter name with spaces in it :)